hivemind/client/expert_uid.py

```diff
@@ -0,0 +1,32 @@
+import re
```

Comment:
I'd argue that expert UID is primarily a server-side issue, but let's merge these two anyway into hivemind.mixture_of_experts in an upcoming PR

Reply:
i respectfully disagree: strictly speaking, hivemind *is* a library for mixture_of_experts. So, its client(for MoE), server(for experts) and averager(for gating function).

If you wish, we can move expert_uid to hivemind.server. Shall we?